pkg/cmd/openshift-controller-manager/cmd.go

```diff
@@ -79,21 +80,17 @@ func (o *OpenShiftControllerManager) Validate() error {
 }
 
 // StartControllerManager calls RunControllerManager and then waits forever
-func (o *OpenShiftControllerManager) StartControllerManager(stopCh <-chan struct{}) error {
-	if err := o.RunControllerManager(); err != nil {
+func (o *OpenShiftControllerManager) StartControllerManager(ctx context.Context) error {
+	if err := o.RunControllerManager(ctx); err != nil {
 		return err
 	}
 
 	go daemon.SdNotify(false, "READY=1")
```

Comment:
we don't use systemd for management anymore, can we remove this?

Reply:
+1 removed